Formula/sshfs.rb

```diff
@@ -13,34 +12,21 @@ class Sshfs < Formula
     sha256 cellar: :any, sierra:      "dc4a7f24c2cbebd7c35891200b043d737ba6586a28992708ef849ffedff7bb01"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on "glib"
-
-  on_macos do
-    deprecate! date: "2020-11-10", because: "requires FUSE"
-    depends_on :osxfuse
-  end
 
-  on_linux do
-    depends_on "libfuse"
-  end
-
-  # Apply patch that clears one remaining roadblock that prevented setting
-  # a custom I/O buffer size on macOS. With this patch in place, it's
-  # recommended to use e.g. `-o iosize=1048576` (or other, reasonable value)
-  # when launching `sshfs`, for improved performance.
-  # See also: https://github.com/libfuse/sshfs/issues/11
-  patch do
-    url "https://github.com/libfuse/sshfs/commit/667cf34622e2e873db776791df275c7a582d6295.patch?full_index=1"
-    sha256 "ab2aa697d66457bf8a3f469e89572165b58edb0771aa1e9c2070f54071fad5f6"
-  end
+  depends_on "glib"
+  depends_on "libfuse"
+  depends_on :linux # Because FUSE must be installed as a cask, and cask dependencies are not allowed
 
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make", "install"
+    mkdir("build") do
```

Comment:
```suggestion
    mkdir "build" do
```
I don't feel too strongly about this, but this is what all the other formulae (with one exception) do, I think.